<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConceptMapEditConceptDialog</class>
 <widget class="QDialog" name="QtConceptMapEditConceptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>766</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>774</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>774</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Concept/Relatie bewerken</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_concept_or_relation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Concept / relatie</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edit_concept">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_examples">
     <property name="text">
      <string>Voorbeelden/Toelichting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_add" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="edit_text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Voeg voorbeeld toe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_bottom" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="toolTip">
         <string>Dit scherm sluiten met verlies van gedane wijzigingen</string>
        </property>
        <property name="text">
         <string>&amp;Afbreken</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ok">
        <property name="toolTip">
         <string>Dit scherm sluiten met behoud van gedane wijzigingen</string>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
